docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,274 @@
+.. _Courses API Courses Resource:
+
+########################################
+Courses API Courses Resource
+########################################
+
+With the Courses API **Courses** resource, you can complete the
+following tasks.
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _Get a list of courses:
+
+**********************
+Get a List of Courses
+**********************
+
+The endpoint to get a list of courses is ``/api/courses/v1/courses/``.
+
+=====================
+Use Case
+=====================
+
+Get a list of the courses that are visible to a specific user. If the request
+is made by an anonymous user, a username is not required.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/courses/?&username=<username>
+
+GET /api/courses/v1/courses/
+
+=====================
+Query Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+* org (optional) - A code for an organization; case-insensitive. Example:
+  "HarvardX". If ``org`` is specified, the list of courses only includes
+  courses belonging to the specified organization.
+
+* mobile (optional) - If specified, the list of courses only includes courses
+  that are designated as ``mobile_available``.
+
+
+=====================
+Response Values
+=====================
+
+The following fields are returned with a successful response.
+
+* blocks_url: Used to fetch the course blocks.
+
+* effort: A textual description of the weekly hours of effort expected in the
+  course.
+
+* end: The date on which the course ends.
```

Comment:
All these date fields are in the [ISO 8601](https://www.w3.org/TR/NOTE-datetime) format.